config/routes.rb

```diff
@@ -43,4 +40,20 @@
       resources :group_assignments, path: 'group-assignments'
     end
   end
+
+  namespace :stafftools do
+    mount Sidekiq::Web  => '/sidekiq'
```

Comment:
I saw that as well, if you think it's ok to use it for just `sidekiq` an nothing else I'm :cool: with that.

If not I can try and explore ways to not use the route constraint